wagtail_wordpress_import/importers/wordpress.py

```diff
@@ -207,6 +218,20 @@ def get_page(self, link, page):
         except self.page_model_instance.DoesNotExist:
             pass
 
+    def connect_page_categories(self, page, category_model, item):
+        categories = item.get("category")
+        if categories:
+            page_categories = []
```

Comment:
**Question**: We reset categories to an empty list first here — would we ever be reimporting the data, or importing categories for an existing page?

Reply:
I made some changes to this function possibly a little more than you mention as I found an edge case where if no categories are attached to a page it could fail.